wolfssl/wolfcrypt/asn.h

```diff
@@ -244,18 +244,19 @@ enum Misc_ASN {
     MAX_SIG_SZ          = 256,
     MAX_ALGO_SZ         =  20,
     MAX_SHORT_SZ        =   6,     /* asn int + byte len + 4 byte length */
-    MAX_SEQ_SZ          =   5,     /* enum(seq | con) + length(4) */
-    MAX_SET_SZ          =   5,     /* enum(set | con) + length(4) */
-    MAX_OCTET_STR_SZ    =   5,     /* enum(set | con) + length(4) */
-    MAX_EXP_SZ          =   5,     /* enum(contextspec|con|exp) + length(4) */
-    MAX_PRSTR_SZ        =   5,     /* enum(prstr) + length(4) */
+    MAX_SEQ_SZ          =   6,     /* enum(seq | con) + length(5) */
+    MAX_SET_SZ          =   6,     /* enum(set | con) + length(5) */
+    MAX_OCTET_STR_SZ    =   6,     /* enum(set | con) + length(5) */
+    MAX_EXP_SZ          =   6,     /* enum(contextspec|con|exp) + length(5) */
+    MAX_PRSTR_SZ        =   6,     /* enum(prstr) + length(5)     */
     MAX_VERSION_SZ      =   5,     /* enum + id + version(byte) + (header(2))*/
     MAX_ENCODED_DIG_ASN_SZ= 9,     /* enum(bit or octet) + length(4) */
     MAX_ENCODED_DIG_SZ  =  64 + MAX_ENCODED_DIG_ASN_SZ, /* asn header + sha512 */
     MAX_RSA_INT_SZ      = 517,     /* RSA raw sz 4096 for bits + tag + len(4) */
```

Comment:
Does MAX_RSA_INT_SZ need to increase by 1 here?

Reply:
Yes, it does.
MAX_RSA_INT_SZ is used for the length of array in SetRsaPublicKey. The array is used for output buffer of SetASNIntRSA().  SetASNInt() called by SetASNIntRSA() calls SetLength(). Therefore, we need to increase by 1.